cmd/spire-agent/cli/run/run.go

```diff
@@ -30,6 +30,8 @@ const (
 	defaultDataDir  = "."
 	defaultLogLevel = "INFO"
 	defaultUmask    = 0077
```

Comment:
Hmm I would have expected this to go away? What role does it play now that we have a minimum?

Reply:
Umask can be changed by the operator too right? Does it make sense to do away with the configurable altogether and take whatever we get so long as it's minimum or better? Just feels a little simpler (and more traditional) than what we have now

Reply:
I've pushed a commit that moves in this direction. If the configurable is set, a warning is issued. If the configurable is does not include the minimum bits required, a warning is issued, and it is upgraded before being set. Otherwise, the current umask is checked, and if it does not have the minimum required bits, a warning is issued and the umask is upgraded.